### ISSUE TITLE
How to get the cordinates of prefiction from predict_webcam() Function

### ISSUE THREAD

**Issue by AmirKaseb** [Deci-AI/super-gradients#1712]
### 💡 Your Question

I want to make a real time prediction using predict_webcam() function  , and i want to get the cordinates (bboxes) of the prediction like the position of the center and the lensth and the width of the predicted square , I tried a lot of solution offered here but it didnt give me real time prediction .

**Comment by Louis-Dupont** [comment]
Hi @AmirKaseb 
`model.predict_webcam()` is more of a demo function. You don't have any control but it will show predictions on the video stream.
If you want to customize its behavior, here is a template of a quickstart code you can use: (I modified some lines of code, so there might be small typos)

``` python
import cv2
from super_gradients.common.object_names import Models
from super_gradients.training import models

# Note that currently only YoloX, PPYoloE and YOLO-NAS are supported.
model = models.get(Models.YOLO_NAS_S, pretrained_weights="coco")

# Here I am drawing on the same camera, but you can change.
cap = cv2.VideoCapture(cv2.CAP_ANY)


while True:
    # Read frames from the cameras
    ret, frame = cap.read()

    # Check if frames are successfully captured
    if not ret:
        break

    image_prediction = model.predict([frame])[0] # Run prediction on single batch


    # TODO: Here you can do anything you want with bbox coordinates, ...
    # Check out https://docs.deci.ai/super-gradients/latest/documentation/source/ModelPredictions.html#detect-objects-in-multiple-images

    frame_with_predictions = image_prediction.draw()

    cv2.imshow(f"Stream: ", frame_with_predictions)

    # Check for key press
    if cv2.waitKey(1) & 0xFF == ord("q"):
        break

# Release the video capture objects and close the windows
cap.release()
cv2.destroyAllWindows()
```

I invite you to check this page https://docs.deci.ai/super-gradients/latest/documentation/source/ModelPredictions.html#detect-objects-in-multiple-images to better understand the `model.predict()` output and how to use it here


**Important**: the code I just provided runs on pytorch model, and not on a compiled engine. This will lead to lower inference speed performances (but is still nice if you want a first demo). 
You can check out this page to learn how to export it, and how to use its results https://docs.deci.ai/super-gradients/latest/documentation/source/models_export.html#new-export-api

Hoping this helps!

**Comment by AmirKaseb** [comment]
thank you
